src/Bridge/Symfony/Bundle/DependencyInjection/Compiler/IdentifierNormalizerPass.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Bridge\Symfony\Bundle\DependencyInjection\Compiler;
+
+use Ramsey\Uuid\Uuid;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\Compiler\PriorityTaggedServiceTrait;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Injects identifier normalizers.
+ *
+ * @author Antoine Bluchet <soyuka@gmail.com>
+ *
+ * @internal
+ */
+final class IdentifierNormalizerPass implements CompilerPassInterface
```

Comment:
What about dropping this class for https://symfony.com/blog/new-in-symfony-3-4-simpler-injection-of-tagged-services and bumping the minimal version to 3.4?